src/js/utils/MarathonUtil.js

```diff
@@ -3,6 +3,12 @@ import VolumeStatus from '../constants/VolumeStatus';
 
 function parseApp(app) {
   let {id, tasks = [], container = {}} = app;
+  Object.defineProperty(app, '_type', {
+    enumerable: false,
+    configurable: false,
+    writable: false,
+    value: 'app'
```

Comment:
If we do the check here, we might as well do the Framework and Application check as well here, moving these lines to here: https://github.com/dcos/dcos-ui/pull/1033/files#diff-a44f44c46bf6bdbc44737179ffb8970dR77

This way we don't differentiate in different places about different types